### ISSUE TITLE
Login not redirecting when global tenant is disabled

### ISSUE THREAD

**Issue by cyramica** [floragunncom/search-guard#411]
Using:
```
Elasticsearch 6.0.0
search-guard-6-6.0.0-17.beta1.zip
search-guard-kibana-plugin-6.0.0-beta1
openjdk version 1.8.0_151
Red Hat Enterprise Linux Server release 7.3
```

It appears that when the global tenant is disabled as such in kibana.yml:
`searchguard.multitenancy.tenants.enable_global: false`
then the user is not redirected after login but instead just sees the login page again without any messages.

In the kibana log file the only part that changes is the referer
from `"referer":"http://10.51.8.220:5601/login?nextUrl=%2F"`
to `"referer":"http://10.51.8.220:5601/login"`

If the user removes all parts of the address trailing the port number then the session allows them to get into Kibana.

**Comment by floragunncom** [comment]
Confirmed as bug, will be fixed in the next release, thanks for reporting!

**Comment by jochenkressin** [comment]
Fixed in 6.1.0-8
